cleanlab_studio/internal/api/api.py

```diff
@@ -155,25 +162,44 @@ def get_label_column_of_project(api_key: str, project_id: str) -> str:
     return label_column
 
 
-def download_cleanlab_columns(api_key: str, cleanset_id: str, all: bool = False) -> pd.DataFrame:
+def download_cleanlab_columns(
+    api_key: str,
+    cleanset_id: str,
+    all: bool = True,
+    to_spark: bool = False,
+) -> Any:
     """
     Download all rows from specified Cleanlab columns
 
     :param api_key:
     :param cleanset_id:
     :param all: whether to download all Cleanlab columns or just the clean_label column
-    :return: return (rows, id_column)
+    :return: return a dataframe, either pandas or spark. Type is Any because don't want to require spark installed
     """
     res = requests.get(
         cli_base_url + f"/cleansets/{cleanset_id}/columns?all={all}",
+        params=dict(to_spark=to_spark),
```

Comment:
nit: can we move the `all` query param to this argument as well instead of adding it to the url manually

Reply:
sounds good updated